src/views/v2/recent-past.php

```diff
@@ -71,9 +71,9 @@
 
 		</div>
 
-		<?php $this->template( 'components/after' ); ?>
+		<?php //$this->template( 'components/after' ); ?>
```

Comment:
* :warning: This comment is 67% valid code; is this commented out code?
* :no_entry_sign: No space found before comment text; expected "// $this->template( 'components/after' );" but found "//$this->template( 'components/after' );"

<sub>hash: b1455df86b584695f8ea2cea50174a53</sub>